MvvmCross/Droid/Shared/Presenter/MvxFragmentsPresenter.cs

```diff
@@ -67,6 +67,11 @@ protected virtual void ShowFragment(MvxViewModelRequest request)
             var serializedRequest = Serializer.Serializer.SerializeObject(request);
             bundle.PutString(ViewModelRequestBundleKey, serializedRequest);
 
+            if (request is MvxViewModelInstanceRequest)
+            {
+                Mvx.Resolve<IMvxChildViewModelCache>().Cache(((MvxViewModelInstanceRequest)request).ViewModelInstance);
```

Comment:
Should we cache the result of `Mvx.Resolve<IMvxChildViewModelCache>()` to avoid a lookup each time?